cmd/algoh/main.go

```diff
@@ -99,6 +99,12 @@ func main() {
 	if absPathErr != nil {
 		reportErrorf("Can't convert data directory's path to absolute, %v\n", dataDir)
 	}
+	
+	algodConfig, err := config.LoadConfigFromDisk(absolutePath)
+
+	if err != nil {
```

Comment:
It's ok that you removed this portion off  `initTelemetry`, but now you need to perform the `!os.IsNotExist(err)` test here as well as load the algodConfig from the `config.GetDefault()` if it's not the case.